tests/fsharp/test-framework.fs

```diff
@@ -68,29 +68,11 @@ module Commands =
     let fsc workDir exec (dotNetExe: FilePath) (fscExe: FilePath) flags srcFiles =
         let args = (sprintf "%s %s" flags (srcFiles |> Seq.ofList |> String.concat " "))
 
-#if FSC_IN_PROCESS
```

Comment:
This was an old attempt to host the F# compiler in process, I think by me, there's no use having it now we primarily invoke via `dotnet`